browsers/chrome_android.json

```diff
@@ -0,0 +1,240 @@
+          "status": "retired"
+        },
+        "35": {
+          "release_date": "2014-05-20",
+          "release_notes": "https://chromereleases.googleblog.com/2014/05/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "36": {
+          "release_date": "2014-07-16",
+          "release_notes": "https://chromereleases.googleblog.com/2014/07/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "37": {
+          "release_date": "2014-09-03",
+          "release_notes": "https://chromereleases.googleblog.com/2014/09/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "38": {
+          "release_date": "2014-10-08",
+          "release_notes": "https://chromereleases.googleblog.com/2014/10/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "39": {
+          "release_date": "2014-11-12",
+          "release_notes": "https://chromereleases.googleblog.com/2014/11/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "40": {
+          "release_date": "2015-01-21",
+          "release_notes": "https://chromereleases.googleblog.com/2015/01/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "41": {
+          "release_date": "2015-03-11",
+          "release_notes": "https://chromereleases.googleblog.com/2015/03/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "42": {
+          "release_date": "2015-04-15",
+          "release_notes": "https://chromereleases.googleblog.com/2015/04/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "43": {
+          "release_date": "2015-05-27",
+          "release_notes": "https://chromereleases.googleblog.com/2015/05/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "44": {
+          "release_date": "2015-07-29",
+          "release_notes": "https://chromereleases.googleblog.com/2015/07/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "45": {
+          "release_date": "2015-09-01",
+          "release_notes": "https://chromereleases.googleblog.com/2015/09/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "46": {
+          "release_date": "2015-10-14",
+          "release_notes": "https://chromereleases.googleblog.com/2015/10/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "47": {
+          "release_date": "2015-12-02",
+          "release_notes": "https://chromereleases.googleblog.com/2015/12/chrome-for-android-update.html",
+          "status": "retired"
+        },
+        "48": {
+          "release_date": "2016-01-26",
+          "release_notes": "https://chromereleases.googleblog.com/2016/01/chrome-for-android-update.html",
```

Comment:
There's no blog post here - it seems there wasn't a blog post for this release.

Reply:
I can't find one either. I'm removing this.